javascript/testing/testing-1.md

```diff
@@ -14,3 +25,10 @@ This lesson is going to center around __Jest__. The biggest reasons for this dec
 4. Read and follow the [Using Matchers](https://facebook.github.io/jest/docs/en/using-matchers.html#content) document on the main Jest website.  This one demonstrates some of the other useful functions you can use in your tests.
 5. Watch [this amazing video](https://www.youtube.com/watch?v=URSWYvyc42M) that covers _what_ to test in your codebase.  The video is specifically about testing the Ruby language, but that doesn't matter _at all_.  The concepts here ring true in any language, and luckily Ruby is a clear enough language that you will be able to follow along just fine.  This video might be worth re-visiting after you've done some testing of your own.
 </div>
+
+
+### Knowledge Check
+- <a class = "knowledge-check-link" href = "#Introduction">What is TDD?</a>
+- <a class = "knowledge-check-link" href = "https://facebook.github.io/jest/docs/en/getting-started.html#content">How to set up and use jest?</a>
+- <a class = "knowledge-check-link" href = "http://godswillokwara.com/index.php/2016/09/09/the-importance-of-test-driven-development/">What are the benefits of writing unit tests?</a>
+- <a class = "knowledge-check-link" href = "https://dzone.com/articles/unit-testing-guidelines-what-to-test-and-what-not">What should you test in your codebase?</a>
```

Comment:
I may be missing it, but if we don’t link it in the lesson, we shouldn’t direct them to it in a Knowledge Check.